test/StdChains.t.sol

```diff
@@ -43,11 +43,6 @@ contract StdChainsTest is Test {
         assertEq(getChain("sepolia").rpcUrl, "https://sepolia.infura.io/v3/b9794ad1ddf84dfb8c34d6bb5dca2001");
     }
 
-    function testFuzz_Rpc(string memory rpcAlias) internal {
-        string memory rpcUrl = getChain(rpcAlias).rpcUrl;
-        vm.createSelectFork(rpcUrl);
-    }
```

Comment:
Ah this got accidentally renamed in https://github.com/foundry-rs/forge-std/pull/433. It should be kept and renamed back to `testRpc`, where it's used in the commented out `testRpcs` test below. That test is commented out since testing public RPCs is slow and flaky